### ISSUE TITLE
Model output

### ISSUE THREAD

**Issue by Jmallins** [MestafaKamal/deepspeech-kabyle#4]
Hi,

I was wondering if it was possible to get the models (transcription) output on the test set?

Thank you!

**Comment by MohammedBelkacem** [comment]
@Jmallins what do you mean?

**Comment by Jmallins** [comment]
Hi @mozillakab

Sorry for the typo. What I meant was, after you trained your model, did you evaluate it on a held out test set (which was not trained on)? If so could I get a copy of your models prediction on this test set?

**Comment by MohammedBelkacem** [comment]
Yes, the models are evaluated and results shared somewhere (I think on a close group on mozilla matrix). @MestafaKamal , please any fresh results to share?
